config.yaml

```diff
@@ -1,3 +1,4 @@
+# It is recommended not to change the default collection, nor the bands or columns types.
 default:
```

Comment:
I don't think we need to capture these parameters under a "default" section. We can just let them at the root.